src/Stateless/Enforce.cs

```diff
@@ -7,13 +7,6 @@ namespace Stateless
 {
     static class Enforce
```

Comment:
I had a weird error message after I deleted the class, the compiler could not find Enforce.Equals or something...
It doesn't complain now, though so I'll get rid of it.